compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -102,18 +99,6 @@ def __init__(
         self._task_status_table: t.Dict[str, t.Dict] = {}
         self.funcx_home = os.path.expanduser(funcx_home)
 
-        for arg, name in [
-            (openid_authorizer, "openid_authorizer"),
-            (fx_authorizer, "fx_authorizer"),
-            (search_authorizer, "search_authorizer"),
-        ]:
-            if arg is not None:
-                msg = (
-                    f"The '{name}' argument is deprecated. "
-                    "It will be removed in a future release."
-                )
-                warnings.warn(msg)
-
```

Comment:
I originally left it in place with something like your suggestion, but `mypy` wasn't happy with unpacking over an empty iterable. I erred on the side of atomicity in commits (for the sake of future `git bisect` users) and decided to temporarily remove it rather than spend too much time trying to make it work.